app/components/image_attachment_table_component.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="card">
+  <!-- table -->
+  <div class="table-responsive overflow-y-hidden">
+    <table class="table mb-0 text-nowrap table-hover table-centered">
+      <thead>
+        <tr>
+          <th scope="col">Image</th>
+          <th scope="col">Filename</th>
+          <th scope="col"></th>
+        </tr>
+      </thead>
+      <tbody>
+        <% if images.any?  %>
+          <% images.each do |image| %>
```

Comment:
could be `Array(images).each do |image| `   and save the outer conditional.

Reply:
As the code currently stands, I think the outer conditional needs to stay since it has an else. I appreciate you pointing out this technique though as I have not seen it before.